src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-
+#odo.uri=${NODO_URI}
```

Comment:
Is it intended to leave these commented out?

Reply:
absolutely no, thanks!